### ISSUE TITLE
Phonetically Unique Words/Phrases

### ISSUE THREAD

**Issue by zctinker** [gryphonshafer/Quizzing-Rule-Book#35]
When considering question writing, questions writers should be aware that quizzers cannot see the words as written in the question and only hear the words. For questions that need have a unique phrase in the first five words, these should not just be as written, but also has heard. 

Example: Q: The sun what?      A: Stopped shining.     R: Luke 24:45
Example: Q: The Son what?     A: Can do nothing by Himself, He can do only what He sees His Father doing.   R: John 5:19

I know these are in different books, but I have seen questions written this way. 

Weigh/Way Son/Sun etc

**Comment by scottpeterson** [comment]
homonyms are only relevant when comparing two questions of the exact same type with the exact same audible question content.  SO in the examples above "the sun stopped what?" would be valid, as would "The Son can do nothing how?"

Often times homonyms exist, but additional words make the phrase unique.

An example of what should be considered Invalid because it's "tricky" is:
CR, Hebrews 11, Passed what? "Past what?

**Comment by zctinker** [comment]
I get that, and yes that would be a better example. I was struggling to remember some of the ones I've seen in the past. Every year there are a handful that I remove from our sets.

**Comment by scottpeterson** [comment]
I don't think there's currently a problem considering these invalid because they're "tricky."

That said, I do think some objective guidelines could be written to make it more clear.

In my hebrews 11 example, if one of those was from a different chapter, then BOTH become valid. It's the Type + Ref + Question pairing that makes it invalid.

**Comment by kclimenhaga** [comment]
I agree that this should be best practice for question writers. In terms of how to incorporate this into the rule book, should we add an example under invalid questions? As well, what do we do about cases like "What route" from Matthew 2? I would normally pronounce that the same way as "What root" from Matthew 3, but I know most Americans would make route rhyme with out instead. Is this something we need to clarify in the rule book? Is that the only example?

**Comment by scottpeterson** [comment]
This feels more like a Question Writing Best Practice, than something we need to add to the rule book. It sounds like the rule book already has the necessary language to rule these as invalid when encountered.

**Comment by jswingle** [comment]
Agreed that the rulebook already covers this with "tricky or misleading." I'd also add that it's unreasonable and difficult for a question writer to find every possible instance of homonyms. I would normally _never_ say this, but these rare situations are probably best left for the quizzers to identify and challenge whenever they find a question to be confusing.

Obviously, if they ARE discovered in the question writing/editing process, they should be changed or deleted.

**Comment by scottpeterson** [comment]
Key wording being "in every way"  :-)

Point of jump is not a factor. Just the question as written.

**Comment by jswingle** [comment]
Right.

"According to John chapter 1, what son?"
"According to John chapter 2, what sun?"

are _not_ identical "in every way".

**Comment by scottpeterson** [comment]
"According to John 1:1, what sun?"
"According to John 1:2, what Son of God?"

also not identical.  Could be based off jump point, but that isn't a factor.

**Comment by jttower** [comment]
The Rules Committee discussed this issue on 9/11/22 and would like to add an example under 2.2.3. Invalid Questions regarding homophones falling under "ambiguous, tricky, or misleading".  I suggest that we change the order so that we can add the example under that bullet point.  
Current:
2.2.3. Invalid Questions
Questions must be declared invalid by the quizmaster or answer judge(s) if:

- The answer requires interpretation by the quizzer
- The question is ambiguous, tricky, or misleading
- The question is multiple-choice
- If the question is not answered
When a question is deemed invalid and thrown out, it must be replaced with the exact same type of question. For example, a CVRMA must be replaced with an CVRMA, not any multiple-answer reference question.
-------------------
Proposed: 
2.2.3. Invalid Questions
Questions must be declared invalid by the quizmaster or answer judge(s) if:

- The answer requires interpretation by the quizzer
- The question is multiple-choice
- The question asked is not answered
- The question is ambiguous, tricky, or misleading. This includes homophones such as Mary's what?/Marries whom?, Passed where?/Past what?, Son/sun, route/root, etc. 

When a question is deemed invalid and thrown out, it must be replaced with the exact same type of question. For example, a CVRMA must be replaced with an CVRMA, not any multiple-answer reference question.

**Comment by jswingle** [comment]
I think having an "Examples" section under "Invalid Questions" is probably the best way to format it. We already have a format for this throughout the rest of the rulebook. But yeah, I like @jttower 's examples.

--

[section number, whatever it ends up being] Examples:

As an example of a question being "ambiguous," consider the two questions "Mary's what?" from Luke 1:41 and "Marries whom?" from Luke 16:18. The quizzer should not be expected to distinguish between these two homophones based solely on the interrogative word chosen by the question writer.

As another example, consider the two questions "What route?" from Matthew 2:12 and "What root?" from Matthew 3:10. Depending on pronunciation, these two questions could be impossible for the quizzer to distinguish from one another.

Only the full question is considered when determining whether a question is ambiguous or not. A quizzer who pre-jumps on the phrase "the sun" may think the question is starting with "the Son," but this does not make the question ambiguous.

**Comment by jswingle** [comment]
While we're at it, we should remove the bullet point from this section concerning "multiple-choice." There's no possible way to write a question as a multiple-choice anyways, so this is a pointless thing to keep in the rulebook.

I also think we should remove the word "tricky" from the bullet point, because it's (ironically) an ambiguous word. It should be considered normal for some questions to be "tough" or "hard," but some people might think "tough/hard" = "tricky." I think the two adjectives "ambiguous or misleading" cover the intent well enough on their own.
